nengo_fpga/tests/test_config.py

```diff
@@ -0,0 +1,101 @@
+"""Tests for fpga_config parser"""
+import os
+
+from nengo_fpga.utils import paths
+from nengo_fpga import fpga_config
+from nengo_fpga.fpga_config import _FPGA_CONFIG, FPGA_CONFIG_FILES
```

Comment:
Reminder to make `FPGA_CONFIG_FILES` lower case once we merge in the `lowercase` PR